graphql/schema/mutation.graphql

```diff
@@ -48,19 +48,18 @@ type Mutation {
   ): Int!
 
   # patch
-  enqueuePatch(patchId: String!, commitMessage: String): Patch!
+  enqueuePatch(patchId: String! @requireProjectAccess(permission: PATCHES, access: EDIT), commitMessage: String): Patch!
   """
   setPatchVisibility takes a list of patch ids and a boolean to set the visibility on the my patches queries
   """
   setPatchVisibility(patchIds: [String!]!, hidden: Boolean!): [Patch!]!
```

Comment:
Yes that's one part of it, but I think it will need a unique directive like `@requirePatchOwner` (similar to `@requireCommitQueueItemOwner`). 

I will make a tech-debt ticket for improving how we handle parameter arrays in general though! 🙂